builder/role_image_test.go

```diff
@@ -239,6 +256,40 @@ func TestGenerateRoleImageDockerfileDir(t *testing.T) {
 
 	// job.MF should not be there
 	assert.Error(util.ValidatePath(filepath.ToSlash(filepath.Join(dockerfileDir, "root/var/vcap/jobs-src/tor/job.MF")), false, "job manifest file"))
+
+	// And verify the config specs are as expected
+	jsonPath := filepath.Join(dockerfileDir, "root/var/vcap/jobs-src/new_hostname/config_spec.json")
+	buf, err := ioutil.ReadFile(jsonPath)
+	if !assert.NoError(err, "Failed to read new_hostname/config_spec.json %s\n", jsonPath) {
+		return
+	}
+	var result map[string]interface{}
+	err = json.Unmarshal(buf, &result)
+	if !assert.NoError(err, "Error unmarshalling output") {
+		return
+	}
+	assert.Equal(0, len(result["properties"].(map[string]interface{})))
+
+	jsonPath = filepath.Join(dockerfileDir, "root/var/vcap/jobs-src/tor/config_spec.json")
+	buf, err = ioutil.ReadFile(jsonPath)
+	if !assert.NoError(err, "Failed to read tor/config_spec.json %s\n", jsonPath) {
+		return
+	}
+	err = json.Unmarshal(buf, &result)
+	if !assert.NoError(err, "Error unmarshalling output") {
+		return
+	}
+
+	var expected map[string]interface{}
+	err = json.Unmarshal([]byte(`{
+		   "tor": {
+            "client_keys": null,
+            "hashed_control_password": null,
+            "hostname": "localhost",
+            "private_key": null
+        }
+	}`), &expected)
+	assert.NoError(err, "Failed to unmarshal expected data")
```

Comment:
Shouldn't we _do_ something with `expected`? Or re-marshal the correct subset of `result` and use `assert.JSONEq` on it?